Apps/W1/ShopifyConnector/app/Base/Page Extensions/ShpfySalesMarketingRC.PageExt.al

```diff
@@ -0,0 +1,75 @@
+/// <summary>
```

Comment:
please remove this RC, we don't support it.

Reply:
Updated with commit "Object Renaming"